swift/apple/FirezoneNetworkExtension/Adapter.swift

```diff
@@ -285,6 +287,8 @@ extension Adapter {
         self.state = .startingTunnel(
           session: try WrappedSession.connect(
             controlPlaneURLString, token, self.getDeviceId(), logDir,
+            // TODO: make logging string depend on build config #2067
+            "connlib_android=info,firezone_tunnel=info,libs_common=info,firezone_client_connlib=info,warn",
```

Comment:
Might be a good idea to add a comment with a link to the tracing documentation for how this works in case the app developers need to change it for debugging purposes.

Reply:
I'm guessing it should be `connlib_apple` instead of `connlib_android`.

Is the log-level meant to be hardcoded to `info`, or is the app expected make it `debug` under certain circumstances (debug build / command-line arg) in a future PR?

Reply:
I think exactly what logs we want to have for debug/release depends on fine tuning but I'd start

For release:

`"connlib_apple=info,firezone_tunnel=info,libs_common=info,firezone_client_connlib=info,error"`

For debug:

`"connlib_apple=debug,firezone_tunnel=trace,libs_common=debug,firezone_client_connlib=debug,warn"`